icu4c/source/i18n/unicode/usearch.h

```diff
@@ -280,9 +280,15 @@ typedef enum {
 /* open and close ------------------------------------------------------ */
 
 /**
-* Creating a search iterator data struct using the argument locale language
+* Create a String Search iterator data struct using the argument locale language
```

Comment:
While and where you change docs please use third person "Creates" etc.

Reply:
Thanks, will rephrase things.